.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+If you haven't done so already, please read the
+[CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md)
+to learn how we work and what we expect from all contributors.
+
+In order to help reviewers, it is always good have some context about your
```

Comment:
I liked it. I'm just not sure if this paragraph shouldn't be in the [CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md) and in the PR template just remember that before to submit a PR, the contributor should actually read the ```CONTRIBUTING.md``` and make sure that tests are passing before open a PR (unless it's a WIP) ?

Reply:
For the tests, it's already addressed in the [CONTRIBUTING.md](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md#mind-the-automated-reviews).

As for this paragraph, I will move it to the CONTRIBUTING.md file.